ui/lerna.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "0.1.2",
+  "npmClient": "yarn",
+  "tagVersionPrefix": "v-ui",
+  "packages": [
```

Comment:
Should we try `useWorkspaces: true` here so that it uses the existing yarn workspace and its config? 

If we do that, then we might have to add `private: true` to the `app/*` packages' `package.json`, but it shouldn't be a problem, I guess.

Reply:
`useWorkspaces: true` was initially part of the options but I had to remove it because Lerna also published everything in the "packages/app/*" dir.

Reply:
Yeah it would, but `private: true` on those packages should prevent that.

Reply:
Also, if we don't include `app/*` packages in `lerna`'s config, then the version bump of the packages in `shared/*` won't get updated on `app/*` projects' dependencies, I guess. Was that bit working fine?

Just curious, I could be wrong too.

Reply:
Interesting. I\ll give that a try and let you know